bootstrapper/internal/kubernetes/kubernetes.go

```diff
@@ -393,13 +388,25 @@ func getIPAddr() (string, error) {
 
 // setupExtraVals create a helm values map for consumption by helm-install.
 // Will move to a more dedicated place once that place becomes apparent.
-func (k *KubeWrapper) setupExtraVals(ctx context.Context, initialMeasurementsJSON []byte, idkeydigest []byte, measurementSalt []byte, subnetworkPodCIDR string, cloudServiceAccountURI string) (map[string]any, error) {
+func (k *KubeWrapper) setupExtraVals(ctx context.Context, serviceConfig constellationServicesConfig) (map[string]any, error) {
+	loadBalancerIP := serviceConfig.loadBalancerIP
+	var err error
+	if strings.Contains(serviceConfig.loadBalancerIP, ":") {
+		loadBalancerIP, _, err = net.SplitHostPort(loadBalancerIP)
+		if err != nil {
+			return nil, fmt.Errorf("splitting host port: %w", err)
+		}
+	}
```

Comment:
nit: since the struct field is called `loadBalancerIP`, conversion from endpoint to IP should happen when creating the struct, not when extracting from it.